module-4/backend/week_one.md

```diff
@@ -5,18 +5,47 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+ --Learning about `this` was super important.
+ 
 2. What are some tools to help debug JavaScript code?
+ --The `debugger` and just go ahead and `console.log()` everything!
+ 
 3. What are some tools you need in order to unit test your JavaScript?
+ --Mocha and Chai, and now Selenium.
+ 
 4. What is the syntax for invoking a function?
+ --Don't forget the parentheses after the name. Pass in any parameters you'll need.
+ 
 5. What is CORS?
+ --Cross Origin Resource Sharing. I think it's referencing the use of resources from other domains. Grabbing stuff to bring render on the DOM.
+ 
 6. How do we enable CORS?
+ --I don't know off the top of my head.
+ 
 7. What's `this` in JavaScript?
+ --It's a contextual object that contains stuff based on where it was invoked.
+ 
 8. What is Webpack and why is it useful?
+ --Webpack grabs all your assets and js and puts them together into a neat little package to send over the DOM.
+ 
 9. When/why do you want to use event delegation?
+ --Event delegation is used for having listeners on things added dynamically to the DOM. If a user could add 100 new things, and you want to listen for action on those things, you wouldn't want a new listener for each thing. Instead, put the listener on a parent element that's present when the DOM is loaded.
+ 
 10. What is a callback function?
+ --More or less a function that executes when after another one.
```

Comment:
Specifically, a second function passed to a first function and invoked by the first function